### PR TITLE
✨ migrate assets store

### PR DESCRIPTION
**Pull request by roiLeo**
## PR Type

- [x] Refactoring

## Context

- [x] related link #4755

## Screenshot 📸

⏳ 

### <samp>🤖 Generated by Copilot at 8e8f487</samp>

> _We used to rely on Vuex_
> _To manage our assets and use_
> _But now we have Pinia_
> _Which makes our code cleaner_
> _And gives us more features to choose_

### CODE REVIEW

**Comment by yangwao** [comment]
@roiLeo what's status on this? I've seen it here for while.

**Comment by roiLeo** [comment]
> @roiLeo what's status on this? I've seen it here for while.

I need #5400 to be done before

**Review by Jarsen136 — APPROVED** [review]
code lgtm

**Comment by vikiival** [comment]
Canary
Screenshot 2023-04-24 at 16 18 36


THis PR;
Screenshot 2023-04-24 at 16 18 22

Looks like assets have disappeared 👨‍🏫

**Comment by roiLeo** [comment]
wfm

Screenshot 2023-05-22 at 14-47-47 KodaDot - NFT Market Explorer

**Comment by vikiival** [comment]
@brano-hozza can I ask you for a review? 🥺

**Comment by roiLeo** [comment]
Deploy failed?

File: stores/assets.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { defineStore } from 'pinia'
+import { AssetListQueryResponse } from '@/components/bsx/Asset/types'
+import assetListQuery from '@/queries/subsquid/bsx/assetList.graphql'
+import { chainAssetOf } from '@/utils/config/chain.config'
+import { useApollo } from '@/utils/config/useApollo'
+import { emptyObject } from '@kodadot1/minimark/utils'
+
+// export type TokenState = ReturnType<typeof state>
```
Comment (brano-hozza) [inline stores/assets.ts:8]: Remove this comment

**Review by brano-hozza — CHANGES_REQUESTED** [review]

**Comment by brano-hozza** [comment]
https://github.com/kodadot/nft-gallery/pull/6070 my suggestion to refactor useApollo so we can remove unnecessary `any` from the code

File: stores/assets.ts (hunk 1 above)
Comment (roiLeo) [inline stores/assets.ts:8]:
```suggestion
```

**Comment by yangwao** [comment]
image

looking for this boi 👀

**Comment by roiLeo** [comment]
for Deepsource issue see #6070 

but for test maybe #6101?
Tested on `/snek/gallery/2773267381-1`
Screenshot 2023-05-25 at 13-58-05 #01 Window KodaDot - NFT Market Explorer

**Comment by roiLeo** [comment]
> @roiLeo it doesnt fix the issue with missing Apollo dependency in the `useNuxtApp`. This is just workaround

maybe it will be fixed when updating to Nuxt3 and @nuxtjs/apollo@next

**Review by vikiival — APPROVED** [review]
Looks oki to me

**Comment by vikiival** [comment]
>  This is just workaround

I'll just ask by meme

7n51qw

**Pull request merged**